examples/fluids/src/setupts.c

```diff
@@ -477,12 +478,17 @@ PetscErrorCode TSSolve_NS(DM dm, User user, AppCtx app_ctx, Physics phys, Vec *Q
     }
   } else {  // continue from time of last output
     PetscReal   time;
-    PetscInt    count;
+    PetscInt    count, step_number;
     PetscViewer viewer;
 
     PetscCall(PetscViewerBinaryOpen(comm, app_ctx->cont_time_file, FILE_MODE_READ, &viewer));
     PetscCall(PetscViewerBinaryRead(viewer, &time, 1, &count, PETSC_REAL));
+    PetscCall(PetscViewerBinaryRead(viewer, &step_number, 1, &count, PETSC_INT));
```

Comment:
@jrwrigh We could put the time and step number into the same binary as the vector, which would simplify the interface. Would you want that done now?